### DISCUSSION TITLE
SQL function in INSERTs

### DISCUSSION THREAD

**Discussion by janekolszak** [aarondl/sqlboiler#1190] [Q&A] [answered]
Hi,
Is it possible to write a query that uses SQL functions with the sqlboiler's query modifiers?
 
`INSERT INTO testgeoms (geom) VALUES (ST_PolygonFromText('POLYGON((5 3, 5 0, 7 0, 7 3, 5 3))'));`

**Comment by stephenafamo** [comment, marked as answer]
No way to do this currently with the querymods.
